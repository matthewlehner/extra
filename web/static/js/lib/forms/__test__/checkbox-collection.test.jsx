// @flow

import React from "react";
import renderer from "react-test-renderer";

import CheckboxCollection from "../checkbox-collection";

describe("CheckboxCollection component", () => {
  const options = [{ value: "0", label: "matthewpearse", provider: "twitter" }];

  it("renders correctly", () => {
    const tree = renderer.create(
      <CheckboxCollection options={options} />
    );

    expect(tree).toMatchSnapshot();
  });
});

