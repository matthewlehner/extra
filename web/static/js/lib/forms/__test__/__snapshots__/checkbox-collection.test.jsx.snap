// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckboxCollection component renders correctly 1`] = `
<div
  className="form__collection-wrapper"
>
  <label
    htmlFor="matthewpearse-0"
  >
    <input
      className="form__control"
      id="matthewpearse-0"
      type="checkbox"
    />
    <span
      className="form__indicator form__indicator_checkbox"
    >
      <svg
        className="icon icon_checkmark"
      >
        <use
          xlinkHref="/images/app.svg#checkmark-icon"
        />
      </svg>
    </span>
    matthewpearse
  </label>
</div>
`;
